### ISSUE TITLE
scoop + git + monokai + vscode = ugly output from git

### ISSUE THREAD

**Issue by kevinpeno** [lukesampson/concfg#65]
I am using concfg and git from scoop, vscode is installed at the user level (not via scoop). I run `concfg import monokai` and everything works fine in powershell. However, when combine with vscode's built in terminal (which also uses powershell) a simple `git status` will kill the theme.

Strangely, it seems to be theme specific whether this error will appear. For example, solarized-dark does not suffer from this issue. Maybe some configs are missing from some themes? Maybe the themes randomly conflict with vscode themes (I'm using the default dark theme in vscode)? Not sure, but I figured I'd try posting about it here first.

**Comment by h404bi** [comment]
Is there any screenshot about the issue you could provided?

**Comment by kevinpeno** [comment]
Sure. Here's solarized-dark: https://prnt.sc/mtqanv
And monokai: https://prnt.sc/mtqavy

**Comment by kevinpeno** [comment]
Following up on this: I don't know why I didn't try this before, but I've pretty much confirmed that there is some issue/conflict between the vscode theme and concfg themes. When I change the vscode theme from the default "dark" to Monokai and set the concfg theme to monokai it works great.

The question now is perhaps, should I file this bug with vscode or do you think this is still somewhat related concfg?

**Comment by h404bi** [comment]
I can't reproduce the issue in my vscode, for me, the powershell which is running in vscode's terminal always uses vscode's theme, no matter what concfg theme I imported.

**Comment by kevinpeno** [comment]
Alright, I think I've figured this out. I'm guessing you have the option "Use legacy console" checked on powershell? After I set that option vscode "owns" the theme and no longer seems to take external inputs like `concfg` into account. `concfg` then only seems to effect appearance when I'm using powershell proper. Perhaps if you disable that you'll see the issue I'm facing, but it might be moot at that point.

Thanks for the back and forth 👍
